book/doctrine.rst

```diff
@@ -641,6 +641,11 @@ to easily fetch objects based on multiple conditions::
 
     If you click the icon, the profiler will open, showing you the exact
     queries that were made.
+    
+    If you make more than 50 queries on a page, the icon will turn yellow to
+    let you know that you are making a lot of requests to the database. This
+    is only a warning and does not have any effect on the execution of further
+    queries.
```

Comment:
well, they should be careful if it goes higher (especially if it happens often). Note that 50 is quite arbitrary here. Blackfire may turn the corresponding metric into red on a different threshold for instance.
the general rule is that lots of DB queries is a bad sign for perf (because DB queries take time)